FileIO/CsvInterface.cpp

```diff
@@ -74,12 +74,13 @@ int CsvInterface::readPoints(std::string const& fname, char delim,
 
 	std::string line;
 	getline(in, line);
-	std::array<std::size_t, 3> const column_idx = 
-		{ CsvInterface::findColumn(line, delim, x_column_name),
-		  CsvInterface::findColumn(line, delim, y_column_name),
-		  (z_column_name.empty()) ? CsvInterface::findColumn(line, delim, y_column_name) : 
-		                            CsvInterface::findColumn(line, delim, z_column_name) };
-	
+	std::array<std::size_t, 3> const column_idx =
+		{{ CsvInterface::findColumn(line, delim, x_column_name),
+			CsvInterface::findColumn(line, delim, y_column_name),
```

Comment:
i mean you should format the arrays as they were before with all array-components starting at the same row (without additional tabs for the 2nd and 3rd component)